### PR TITLE
Clean up MultipleConnectAsync and NetworkStream.

### PR DESCRIPTION
**Pull request by pgavlin**

### CODE REVIEW

**Comment by pgavlin** [comment]
@davidsh @stephentoub PTAL

**Comment by davidsh** [comment]
LGTM

File: src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs [hunk 1]
```diff
@@ -8,49 +8,27 @@
...
-        /// </devdoc>
+        // Used by the class to indicate that the stream is writable.
         private bool _writeable;
 
         private bool _ownsSocket;
 
-        /// <devdoc>
-        /// <para>Creates a new instance of the <see cref='System.Net.Sockets.NetworkStream'/> without initalization.</para>
-        /// </devdoc>
+        // Creates a new instance of the System.Net.Sockets.NetworkStream without initalization.
```
Comment (stephentoub) [inline src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs:25]: Do you want to still use XML comments so they show up elsewhere in the solution in Intellisense, can be used in doc generation, etc.?

**Comment by stephentoub** [comment]
LGTM

File: src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs (hunk 1 above)
Comment (pgavlin) [inline src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs:25]: I'm putting this off until we decide on a documentation strategy for System.Net. The edits done here are consistent with what was done in other S.N assemblies.

**Comment by CIPop** [comment]
LGTM

**Pull request merged**